test/precompile/hedera-token-service/utils.js

```diff
@@ -395,6 +494,64 @@ class Utils {
     return tokenAddress;
   }
 
+  static hexToASCII(str) {
```

Comment:
There is a similar method defined here https://github.com/hashgraph/hedera-smart-contracts/blob/main/test/precompile/hedera-token-service/redirect-for-token.js#L40. We may consider extracting these methods in utils class?